pbclient/__init__.py

```diff
@@ -454,6 +454,34 @@ def get_tasks(project_id, limit=100, offset=0, last_id=None):
         raise
 
 
+def get_new_task(project_id, limit=1, offset=0, last_id=None):
```

Comment:
I decided to drop params: limit, offset and last_id. These parameters are relevant only for getting multiple tasks. It just doesn't apply when you want to get a random task.

Reply:
I need your opinion on this one: is it desired use case to send multiple new tasks at once?

Reply:
Yes. It will be nice to add it. Basically think about this of getting a bunch of tasks for your user with only one request. It will save time and you can cache them.